### PR TITLE
Add Obsoletes of all the rails-packaging versions

### PR DESCRIPTION
**Pull request by zjhuntin**

### CODE REVIEW

File: packages/foreman/rubygem-actionview/rubygem-actionview.spec [hunk 1]
```diff
@@ -34,6 +34,8 @@ BuildArch: noarch
 Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
 # end specfile generated dependencies
 
+Obsoletes: tfm-ror52-rubygem-%{gem_name} <= 5.2.1
```
Comment (ekohl) [inline packages/foreman/rubygem-actionview/rubygem-actionview.spec]: Looks like you forgot to bump this one

File: packages/foreman/rubygem-multi_json/rubygem-multi_json.spec [hunk 2]
```diff
@@ -31,6 +31,8 @@ BuildRequires: %{?scl_prefix_ruby}ruby(rubygems)
 BuildArch: noarch
 Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
 
+Obsoletes: tfm-ror52-rubygem-%{gem_name} <= 1.12.2
```
Comment (ekohl) [inline packages/foreman/rubygem-multi_json/rubygem-multi_json.spec]: Missed the bump here

**Comment by ehelms** [comment]
Adding the collection to here https://github.com/theforeman/forklift/blob/master/bats/fb-verify-packages.bats#L15 as part of this getting merged will be useful for catching any missing packages.

**Comment by evgeni** [comment]
the passenger repoclosure issue *seems* unrelated?

**Comment by ekohl** [comment]
Is that because of the version mismatch in our repo and the one in EPEL?

**Comment by ehelms** [comment]
The current repoclosure error is just nightly pipeline needing to run and update. So yes, it is unrelated.

**Pull request merged**